### PR TITLE
Update finance.yml

### PR DESCRIPTION
**Pull request by JMCARY**
Updated Northwestern Mutual to use software token, sms, and phone, no public facing documentation available

### CODE REVIEW

File: _data/finance.yml [hunk 1]
```diff
@@ -335,7 +335,10 @@ websites:
 
   - name: Northwestern Mutual
     url: https://www.northwesternmutual.com/
-    facebook: northwesternmutual
+    tfa:
+      - sms
+      - phone
+      - totp
     img: northwesternmutual.png
```
Comment (jdavis) [inline _data/finance.yml]:
```suggestion
    doc: https://www.northwesternmutual.com/security-and-privacy/
    
```

**Review by jdavis — CHANGES_REQUESTED** [review]
Maybe we should use this for docs? It does at least mention 2FA is supported: https://www.northwesternmutual.com/security-and-privacy/

**Review by jdavis — APPROVED** [review]

**Comment by jdavis** [comment]
Thanks John!

**Comment by JMCARY** [comment]
Thank you. I use your site. Had admired from a distance. Would like to contribute in other ways not an app dev.

________________________________

**Pull request merged**